compiler/rustc_session/src/config.rs

```diff
@@ -2135,11 +2116,6 @@ fn check_error_format_stability(
     }
     let format = match format {
         ErrorOutputType::Json { pretty: true, .. } => "pretty-json",
-        ErrorOutputType::HumanReadable { kind, .. } => match kind {
-            HumanReadableErrorType::AnnotateSnippet { unicode: false, .. } => "human-annotate-rs",
```

Comment:
I would make the removal of the `human-annotate-rs` flag its own commit, although I am not sure if it should be before or after the commit switching to `annotate-snippets` as the default. 

Note: That change will also require removing any references to it